### ISSUE TITLE
T4 p2 error pygame

### ISSUE THREAD

**Issue by vice-rosas99** [IIC2613-Inteligencia-Artificial-2022-2/Syllabus#97]
al intentar correr el archivo qagent.py me sale este error causado por el juego
![image](https://user-images.githubusercontent.com/62083572/204674020-f203196d-1012-4669-a251-437ff1805953.png)

Como puedo arreglar esto?

**Comment by sebaterrazas** [comment]
Me pasó lo mismo, tienes que correr el arhivo de `QAgent.py` en la misma carpeta en donde está. Osea cámbiate al directorio donde está, con `cd` por ejemplo, y después corre el código.

**Comment by dfloreaa** [comment]
Efectivamente, el programa no encuentra el directorio para los recursos gráficos porque se encuentra trabajando desde otra carpeta base. Se encuentra pusheada en el repositorio de la tarea una nueva versión que soluciona esto y permite correr el archivo desde cualquier carpeta, mil disculpas por ello.

En todo caso, aquí están las líneas que se deben cambiar:

# Añadir estas líneas
import os
current_path = os.path.dirname(os.path.realpath(__file__))

# Reemplazar las líneas 27-35 de FlappyBirdAI.py por estas:
SCORE_FONT = pygame.font.Font(os.path.join(current_path, "resources/fb-font.ttf"), 80)
SPRITE = pygame.image.load(os.path.join(current_path, "resources/sprite.png"))
BOT_PIPE = pygame.image.load(os.path.join(current_path, "resources/bot_pipe.png"))
TOP_PIPE = pygame.image.load(os.path.join(current_path, "resources/top_pipe.png"))
BACKGROUND = pygame.image.load(os.path.join(current_path, "resources/background.png"))
FLOOR = pygame.image.load(os.path.join(current_path, "resources/floor.png"))
